datatypes/blueprints/datatypes.py

```diff
@@ -16,50 +17,55 @@
 
 @bp.route('/', methods=['GET'])
 @cross_origin()
-def get_all_datatypes():
+def get_all_data_types():
     db = get_database()
-
-    log.info('Fetching all datatypes')
-    script = get_script_path(SELECT_ALL_DATA_TYPES)
-    datatypes = db.execute_script(script)
-
-    if not datatypes:
-        log.info('No entries found.')
-        return responding.no_content(datatypes)
-
-    log.debug('Entries: {}'.format(datatypes))
-    return responding.ok(datatypes)
-
-
-@bp.route('/uuidrefs', methods=['GET'])
-@cross_origin()
-def get_all_refs():
-    db = get_database()
-    req_datatype = request.args.get('datatype')
+    req_data_type = request.args.get('datatype')
     req_date = request.args.get('date')
-    if not req_datatype:
-        log.debug('No query parameter was passed to the method. Returning empty')
-        return responding.no_content([])
+    limit = request.args.get('limit') or 10
+    offset = request.args.get('offset') or 0
+
+    if not req_data_type:
+        log.info('Fetching all data types')
+        fields = {}
+        script = get_script_path(SELECT_ALL_DATA_TYPES)
 
-    if req_datatype and req_date:
+    elif req_data_type and req_date:
+        log.info('Fetching by type ({}), limited by date ({})'.format(req_data_type, req_date))
+        fields = {'datatype': req_data_type, 'anydate': req_date, 'limit': limit, 'offset': offset}
         script = get_script_path(SELECT_BY_DATA_TYPES_ON_DATE)
-        datatype = {'datatype': req_datatype, 'anydate': req_date}
-        log.info('Fetching ref uuids for datatype, anydate: {}'.format(datatype))
-        entities = db.execute_script(script, **datatype)
 
     else:
+        log.info('Fetching by data type: {}'.format(req_data_type))
```

Comment:
Might be something I overlook, but why are you using `'hello {}'.format('world')` instead of `logger.info('hello %s', 'world')`?

Reply:
`.format` is used across all of sqapi. This might be changed to using f-strings in the future :-)